### ISSUE TITLE
Step sequencer support

### ISSUE THREAD

**Issue by 2b2a2t2** [midilab/aciduino#36]
Hello !
Would there be a simple way to add (in the setup file for example) a physical support for the 16 step sequencer (using a 4067 mux or several MPR121), we could declare an array for the step buttons independently of the hardware ?
the same for LEDs could we use WS2812 (one more cable per LED but uses only one GPIO pin)
thanks !

**Comment by midilab** [comment]
Hi @2b2a2t2 

> Would there be a simple way to add (in the setup file for example) a physical support for the 16 step sequencer (using a 4067 mux or several MPR121), we could declare an array for the step buttons independently of the hardware ?

Yes, this is the  generic 16 button/LED extension module to be released, there is already some code implemented on Aciduino, but it is not yet complete, but it follows the same idea of the 16-knob extension released, so now this module is already on the roadmap. 

For the CIs, aciduino is based on uCtrl framework wich has his list of CIs and drivers avaliable to use, please take a look at the [documantion of uCtrl](https://github.com/midilab/uCtrl#din). The 4067 is only for Touch buttons(capacitive), for normal button matrix you should go for 74H165 and for the leds the 74H595, any other CIs is not avaliable at the moment and new support requests should be asked on [uCtrl](https://github.com/midilab/uCtrl) project it self.

But if you really want to play around, I have initiated the support for the step sequencer buttons matrix. The [uOne PCB](https://raw.githubusercontent.com/midilab/aciduino/master/v2/hardware/imgs/uone_umodular.jpg) currently utilizes it, although it may not be fully functional as intended since it is still under development.

Basically, I need to complete the user flow definitions for the generic 16-button/LED module for pages other than the step sequencer. Additionally, I need to incorporate shift presses, most likely for track selection.
